packages/auth/src/signature/middleware.ts

```diff
@@ -69,11 +69,12 @@ export async function grantContinueHttpsigMiddleware(
   )
 
   const grantService = await ctx.container.use('grantService')
+  // TODO: if interactRef exists, check that interaction is found
```

Comment:
Did you implement this TODO below, on lines 87-97? This seems like it's doing what the TODO describes so maybe this comment can be removed?

```ts
  const interactionService = await ctx.container.use('interactionService')
  const interaction =
    interactRef && (await interactionService.getByRef(interactRef))

  if (!interaction) {
    ctx.status = 401
    ctx.body = {
      error: 'invalid_continuation',
      message: 'invalid interaction'
    }
  }
```

Reply:
Yeah, I just forgot to clean up the TODO. Thanks for pointing that out.